src/Core/Command/AgentAction.ts

```diff
@@ -0,0 +1,16 @@
+import { Maybe } from "@/Core/Language";
+import { Query } from "@/Core/Query";
+
+export interface AgentAction {
```

Comment:
The naming of this command is not really in line with the other commands.
It should be a "command", a statement, using a verb.
I know AgentAction is the one used in the backend.
But I don't think we have followed this terminology for the other commands.